frontend/packages/console-shared/src/components/badges/Badge.scss

```diff
@@ -5,7 +5,7 @@
   }
 }
 
-.tech-preview {
+.odc-preview-badge {
```

Comment:
```suggestion
.ocs-preview-badge {
```

Reply:
Updated.

Reply:
This is not updated. I can still see the preview-badge class not prefixed with ocs. https://github.com/openshift/console/pull/6250/files#diff-0c1edabb529e4a788e55f04adb9c44a3

Reply:
Had gotten confused with the similar review above, fixed now.